activation/wire/wire_v1.go

```diff
@@ -25,7 +25,7 @@ type ActivationTxV1 struct {
 	blob []byte
 }
 
-// InnerActivationTxV1 is a set of all of an ATX's fields, except the signature. To generate the ATX signature, this
+// InnerActivationTxV1 is a set of all of an ATXs fields, except the signature. To generate the ATX signature, this
```

Comment:
The original comment was a bit more correct, as `all of ATXs fields` with `ATXs` being plural doesn't make much sense ;)
Probably this way it would be better
```suggestion
// InnerActivationTxV1 is a set of all the fields in an ATX, except the signature. To generate the ATX signature, this
```

Reply:
Yeah, the comment is difficult to read. I updated it to make clearer what the purpose of `InnerActivationTxV1` is.